src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -1144,7 +1144,7 @@ TEST_CASE("contract storage", "[tx][soroban]")
         {
             // Only restore WASM
             restoreOp({contractKeys[1]},
-                      87 /* rent bump */ + 20000 /* one LE write */);
+                      86 /* rent bump */ + 20000 /* one LE write */);
```

Comment:
Why is the difference one stroop?

Reply:
Why not? The key is small and the bump period is small.